x-pack/plugins/elastic_assistant/server/lib/attack_discovery/evaluation/example_input/index.test.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ExampleInput, ExampleInputWithOverrides } from '.';
+
+describe('ExampleInput Schema', () => {
+  it('validates a correct ExampleInput object', () => {
+    const validInput = {
```

Comment:
nit: Consider making `validInput` a top level variable and spreading it into the input variables below to make these tests more readable.